pkg/internal/testhelpers/prometheus_container.go

```diff
@@ -0,0 +1,81 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package testhelpers
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/go-connections/nat"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const prometheusImage = "prom/prometheus:main"
```

Comment:
Same as with the jaeger container. However in this case we can use `prom/prometheus:latest` as it always points to the latest tagged release.

Reply:
This was earlier `:latest`, but we wanted the most recent merge with master for promql tests, hence this was done.